### PR TITLE
Stories/983 overlay download free plugin

### PR DESCRIPTION
**Pull request by Sjardo**
Fixes https://github.com/Yoast/yoast.com/issues/983

Goes along with: https://github.com/Yoast/yoast.com/pull/1210

### CODE REVIEW

File: html_includes/partials/modal-free-plugin.php [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+wp_enqueue_script( 'jquery-modal' );
+add_action( 'wp_footer', 'yst_footer_modal_free_plugin_script', 10 );
+
+	?>
+	<div id="free-plugin-download-modal" style="display: none;">
+		<h3>
+			<?php _e( 'Great idea to get our free SEO plugin', 'yoastcom' ); ?>
+		</h3>
+		<div class="content content-downloading grid">
```
Comment (thulshof) [inline html_includes/partials/modal-free-plugin.php]: Please fix indentation.

Comment (thulshof) [inline html_includes/partials/modal-free-plugin.php]: Please fix indentation of this code block.

File: html_includes/partials/modal-free-plugin.php [hunk 2]
```diff
@@ -47,11 +47,11 @@ function position_modal(){
 		</div>
         <div class="content content-usps">
             <p>
-                <strong><?php _e( 'Wait! we get that you\'re thrilled to get started. but wait...', 'yoastcom' ); ?></strong><br/>
+                <strong><?php _e( 'Wait! we get that you\'re thrilled to get started. But wait...', 'yoastcom' ); ?></strong><br/>
                 <?php _e( 'Why not first make sure you get the most out of it?', 'yoastcom' ); ?>
             </p>
             <ul class="list--usp">
-                <li><?php _e( 'Get weekly tips on how to optimize your website\'s SEO, usavility and conversion', 'yoastcom' ); ?></li>
+                <li><?php _e( 'Get weekly tips on how to optimize your website\'s SEO, usavility and conversion.', 'yoastcom' ); ?></li>
```
Comment (thulshof) [inline html_includes/partials/modal-free-plugin.php]: There is a type in `usavility`. Please fix.

**Comment by maartenleenders** [comment]
Acceptance 🚧 

The modal pops up as intended in my browsers, but there are some small typing errors and the copy is nog optimal.

Suggested changes for the copy (in accordance with @Annelieke ):
Current situation:
image

There is a spelling error in `downloadfolder`, but actually we don't know which folder they are going to download to. So we'd like the sentence to be changed to:
`You'll find it on your computer in a few seconds.`

In the sentence `Wait! We get that you're thrilled to get started.........etc`, the `w` in `we` is not capitalized. It should be. Also, the first `Wait!` should be in bold, if possible.

Because you cannot "first" make sure you get the most out of the plugin, before actually using it, we can drop the word `first` from `Why not first make sure you get the most out of it?` to create a sentence that just runs a bit more smoothly.

**Comment by NicolleY** [comment]
Acceptance done 👍

**Pull request merged**